packages/gatsby-plugin-gatsby-cloud/src/components/Indicator.js

```diff
@@ -1,21 +1,246 @@
 import React, { useState, useEffect, useCallback, useRef } from "react"
-
+import { formatDistance } from "date-fns"
 import getBuildInfo from "../utils/getBuildInfo"
 import trackEvent from "../utils/trackEvent"
+import IndicatorButton from "./IndicatorButton"
+import {
+  gatsbyIcon,
+  logsIcon,
+  failedIcon,
+  infoIcon,
+  linkIcon,
+  successIcon,
+} from "./icons"
 import Style from "./Style"
 
-import GatsbyIndicatorButton from "./GatsbyIndicatorButton"
-import LinkIndicatorButton from "./LinkIndicatorButton"
-import InfoIndicatorButton from "./InfoIndicatorButton"
-
 const POLLING_INTERVAL = process.env.GATSBY_PREVIEW_POLL_INTERVAL || 3000
 
-export default function Indicator({ children }) {
+export function PreviewIndicator({ children }) {
+  return (
+    <>
+      <Style />
+      <div
+        data-testid="preview-status-indicator"
+        data-gatsby-preview-indicator="root"
+        aria-live="assertive"
+      >
+        {React.Children.map(children, (child, i) =>
+          React.cloneElement(child, { ...child.props, buttonIndex: i })
+        )}
+      </div>
+    </>
+  )
+}
+
+export function GatsbyIndicatorButton(props) {
+  return <IndicatorButton testId="gatsby" iconSvg={gatsbyIcon} {...props} />
+}
+
+const copySuccessTooltip = (
```

Comment:
React components should start with capital letter
```suggestion
const CopySuccessTooltip = (
```